packages/cmf/src/components/ErrorPanel/ErrorPanel.component.js

```diff
@@ -18,29 +17,14 @@ function ErrorPanel(props) {
 			onError.revokeObjectURL(url);
 		};
 	}, [props.error]);
-	let currentErrorStatus;
-	if (props.reported) {
-		currentErrorStatus = `Has been reported under ${props.response.id}`;
-	} else {
-		currentErrorStatus = get(props.reason, 'message');
-	}
+	const HAS_REPORT = onError.hasReportFeature();
 	return (
 		<div>
 			<p className="error-title">
 				{props.error.name}: {props.error.message}
 			</p>
-			<React.Fragment>
-				{onError.hasReportURL() ? (
-					<p>
-						The error report has been sent.
-						<br />
-						Report status: {currentErrorStatus}
-					</p>
-				) : null}
-				<p>From here you have two choices:</p>
-				<p>1. Refresh the app and retry</p>
-				<p>2. Download the details and then contact the support</p>
-			</React.Fragment>
+			{HAS_REPORT && <p>The error report has been sent.</p>}
```

Comment:
i18n.t here?

Reply:
I need to say no.
CMF is i18n agnostic (today no string is supposed to be displayed to the end user.
I would prefer not to create i18n project just for one string and by the way it will make i18n critic in any case of error.